packages/@react-types/form/src/index.d.ts

```diff
@@ -14,11 +14,21 @@ import {DOMProps, SpectrumLabelableProps, StyleProps, ValidationState} from '@re
 import {ReactElement} from 'react';
 
 export interface SpectrumFormProps extends DOMProps, StyleProps, SpectrumLabelableProps {
+  /** The contents of the form. */
```

Comment:
I remember Danni saying is slack we always want to capitalize component names. We aren't consistent in JSdocs on this.

Reply:
I'll go ahead and change the `form` to `Form` here, will handle the rest in a followup

Reply:
I'll change this for form and handle the rest in a follow up